evap/staff/templates/staff_evaluation_form.html

```diff
@@ -11,7 +11,16 @@ <h3>
     </h3>
     {% if evaluation_form.instance.id %}
         <div class="mb-2">
-            {% include 'course_badges.html' %}
+            {% if evaluation.state == 'new' %}
+                <span class="badge badge-light"><span class="fas fa-circle icon-yellow"></span> {% trans 'In preparation' %}</span>
+            {% elif evaluation.state == 'prepared' %}
+                <span class="badge badge-light"><span class="far fa-square icon-gray"></span> {% trans 'Awaiting editor review' %}</span>
+            {% elif evaluation.state == 'editor_approved' %}
+                <span class="badge badge-light"><span class="far fa-check-square icon-yellow"></span> {% trans 'Approved by editor' %}</span>
+            {% elif evaluation.state == 'approved' or state == 'in_evaluation' or state == 'evaluated' or state == 'reviewed' or state == 'published' %}
```

Comment:
is using only `state` instead of `evaluation.state` correct here?

Reply:
oh boy. it's not - thanks :)